code/modules/fishing/fishing_gear.dm

```diff
@@ -179,6 +193,32 @@
 			src.icon_state = "fishing_rod_3-inactive"
 			src.item_state = "fishing_rod-inactive"
 
+/obj/item/fishing_rod/cybernetic
+	name = "cybernetic fishing rod"
+	desc = "A Cybernetic Fishing Rod. Use on a fishing rod on the ground to upgrade."
+	icon_state = "fishing_rod-inactive"
+	item_state = "fishing_rod-inactive"
+	tier = 1
+
+	update_icon()
+		if ((src.is_fishing) && (src.tier == 1))
+			src.icon_state = "fishing_rod-active"
+			src.item_state = "fishing_rod-active"
+		else if ((src.is_fishing) && (src.tier == 2))
+			src.icon_state = "fishing_rod_2-active"
+			src.item_state = "fishing_rod-active"
+		else if ((src.is_fishing) && (src.tier == 3))
+			src.icon_state = "fishing_rod_3-active"
+			src.item_state = "fishing_rod-active"
+		else if ((!src.is_fishing) && (src.tier == 1))
+			src.icon_state = "fishing_rod-inactive"
+			src.item_state = "fishing_rod-inactive"
+		else if ((!src.is_fishing) && (src.tier == 2))
+			src.icon_state = "fishing_rod_2-inactive"
+			src.item_state = "fishing_rod-inactive"
+		else
+			src.icon_state = "fishing_rod_3-inactive"
+			src.item_state = "fishing_rod-inactive"
```

Comment:
```suggestion
		var/isactive = src.is_fishing ? "active" : "inactive"
		src.icon_state = "fishing_rod[src.tier > 1 ? "_[src.tier]" : ""]-[isactive]"
		src.item_state = "fishing_rod-[isactive]"
```

Giant elif block can be simplified a bit with some code

Reply:
Well, that makes things simpler. Thanks!